File: producer.py
import os
import json
import boto3

sqs = boto3.client('sqs')

def handler(event, context):
    message = json.dumps({ 'text': 'Hello from API Gateway s' })

    sqs.send_message(
        QueueUrl=os.environ['QUEUE_URL'],
        MessageBody=message
    )

    return {
        'statusCode': 200,
        'body': 'Message sent to SQS'
    }

File: producer.js
const AWS = require('aws-sdk');
const sqs = new AWS.SQS();

exports.handler = async (event) => {
  const message = JSON.stringify({ text: 'Hello from API Gateway s' });

  await sqs.sendMessage({
    QueueUrl: process.env.QUEUE_URL,
    MessageBody: message
  }).promise();

  return { statusCode: 200, body: 'Message sent to SQS' };
};
